main.py

```
@@ -14,31 +15,67 @@ def order_menu():
+        else:
+            print("Employee does not exist")
+            return
+        
+        # Add Items to Order_Items
+        while(user_input := input("Would you like to add items to the order? (Y/N) ").upper()) == "Y":
+            ItemID = input("What is the ItemID of the item you would like to add? ")
+            ItemQuantity = input("How many of this item would you like to add? ")
+
+            database.add_items_to_order(orderID, ItemID, ItemQuantity)
+
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)